README.md

```diff
@@ -367,44 +367,46 @@ Special builds:
 | 1.x-centos7 | dockerfile/centos_7 |
 | 2.x-centos7 | dockerfile/centos_7 |
 
-## How to push changes (for maintainers)
+## Release policy
 
-When the change is about specific tarantool version or versions
-range, update all relevant fixed versions & rolling versions
-according to the pipelines listed above.
+All images are pushed to [Docker Hub](docker_hub_tags).
 
-When the change is about the environment at all, all versions
-need to be updated.
+Fixed version tags (`x.y.z`) are frozen: we never update them.
```

Comment:
If we assume there will be hot updates with some suffixes (or numbers?) after the 3rd number, then we should extend definition of fixed tag